announcement/models/announcement.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AnnouncementLog(models.Model):
+    _name = "announcement.log"
+    _description = "Log user reads"
+    _order = "create_date desc"
+
+    announcement_id = fields.Many2one(comodel_name="announcement")
+
+
+class Announcement(models.Model):
+    _name = "announcement"
+    _description = "User announcements"
+    _order = "sequence asc, id"
+
+    active = fields.Boolean()
```

Comment:
```suggestion
    active = fields.Boolean(default=True)
```

Reply:
No. It's on purpose. We don't want announcements released until they're unarchived.